front/app/components/ProjectAndFolderCards/components/Topbar/Tabs.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useRef, KeyboardEvent } from 'react';
+
+// components
+import { ScreenReaderOnly } from 'utils/a11y';
+
+// styling
+import styled from 'styled-components';
+import { fontSizes, isRtl, colors, media } from 'utils/styleUtils';
+import { rgba } from 'polished';
+
+// i18n
+import messages from './messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+// typings
+import { IStatusCounts } from 'hooks/useAdminPublicationsStatusCounts';
+import { PublicationTab } from '../..';
+
+const TabsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+
+  ${media.largePhone`
+    width: 100%;
+    justify-content: space-between;
+  `}
+
+  ${isRtl`
+    flex-direction: row-reverse;
+  `}
+`;
+
+const Tab = styled.button<{ active: boolean }>`
+  box-sizing: content-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-size: ${fontSizes.large}px;
+  padding: 0px 15px;
+
+  border-bottom: ${({ active, theme }) =>
+    active ? `3px solid ${theme.colorMain}` : '3px solid transparent'};
+
+  color: ${({ active, theme }) => (active ? theme.colorMain : colors.label)};
+
+  ${({ active, theme }) =>
+    active
+      ? ''
+      : `
+      &:hover {
+        border-bottom: 3px solid ${rgba(theme.colorMain, 0.3)};
+      }
+      cursor: pointer;
+    `}
```

Comment:
`active && ...` will return `false` if `active === false`. Seems weird to return a boolean in a function that is expected to return a string.

With `if (active)` or `if (!active)`, you will still have to return a string somewhere, otherwise you get the 'not all code paths return a value' error

Reply:
Check VoteButton.tsx for examples of both!